WalletWasabi.Fluent/ViewModels/AddWallet/WalletNamePageViewModel.cs

```diff
@@ -95,7 +95,7 @@ private async Task CreatePasswordAsync(string walletName)
 						Services.WalletManager.WalletDirectories.WalletsDir,
 						Services.WalletManager.Network)
 					{
-						TipHeight = Services.BitcoinStore.SmartHeaderChain.TipHeight
+						TipHeight = Services.BitcoinStore.SmartHeaderChain.ServerTipHeight
```

Comment:
Is this correct?

My understanding is `TipHeight` is the height of the downloaded filters and `ServerTipHeight` is (more or less) the height of the blocks.
Meaning if you create a new wallet there is no chance to have any relevant transaction before `ServerTipHeight`.

@molnard Can you confirm it?

Reply:
Yes, it is true.